src/error_chain.rs

```diff
@@ -68,6 +68,21 @@ macro_rules! impl_error_chain_cause_or_source {
         };
 }
 
+/// Conditional usage of deprecated Error::description
+#[doc(hidden)]
+#[cfg(has_error_description_deprecated)]
+#[macro_export(local_inner_macros)]
+macro_rules! call_to_deprecated_description {
+    ($e:ident) => { "" };
+}
+
+#[doc(hidden)]
+#[cfg(not(has_error_description_deprecated))]
+#[macro_export(local_inner_macros)]
+macro_rules! call_to_deprecated_description {
+    ($e:ident) => { ::std::error::Error::description($e) };
+}
```

Comment:
Doesn't this need a doc comment, too? Wouldn't `#![deny(missing_documentation)]` trigger in case `not(has_error_description_deprecated)`?

Reply:
That's why CI fails.

Reply:
Tested, it fails and reverted it back